android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -84,6 +93,22 @@ public String getName() {
         return map;
     }
 
+    @RequiresApi(Build.VERSION_CODES.O)
+    private void createChannel(ReactApplicationContext context) {
+        NotificationManager
+            mNotificationManager =
+            (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+
+        CharSequence name = "Media playback";
+        String description = "Media playback controls";
+        int importance = NotificationManager.IMPORTANCE_LOW;
```

Comment:
There's no need to split these into variables as they're only being used once

Reply:
Ok, right.